src/components/Body/MainSection.js

```diff
@@ -0,0 +1,55 @@
+function changeDateFormat(date) {
+  const newDate = new Date(date);
+  return newDate.toDateString();
+}
+
+function timeSince(date) {
+  const seconds = Math.floor((new Date() - new Date(date)) / 1000);
+  let interval = seconds / 31536000; // 60 * 60 * 24 * 365
+
+  if (interval > 1) {
+    const years = Math.floor(interval);
+    return years <= 1 ? "1 year ago" : `${Math.floor(years / 12)} years ago`;
+  }
+  interval = seconds / 2592000; // 60 * 60 * 24 * 30
+  if (interval > 1) {
+    return interval <= 1 ? "1 month ago" : `${Math.floor(interval)} months ago`;
+  }
+  interval = seconds / 86400; // 60 * 60 * 24
+  if (interval > 1) {
+    return interval <= 1 ? "1 day ago" : `${Math.floor(interval)} days ago`;
+  }
+  interval = seconds / 3600; // 60 * 60
+  if (interval > 1) {
+    return interval <= 1 ? "1 hour ago" : `${Math.floor(interval)} hours ago`;
+  }
+  interval = seconds / 60;
+  if (interval > 1) {
+    return `${Math.floor(interval)} minutes ago`;
+  }
+  return "1 minute ago";
+}
+
+function CardData({ data }) {
+  const { description, imageSource, createdAt } = data;
+  return (
+    <div className="card-frame">
+      <img className="card-image-size" src={imageSource} alt={imageSource} />
+      <div className="card-description-frame">
+        <p>{description}</p>
+        <p>{changeDateFormat(createdAt)}</p>
+        <p>{timeSince(createdAt)}</p>
+      </div>
+    </div>
+  );
+}
+
+export function MainSection({ data }) {
+  return (
+    <div className="main-website-section">
+      {data.map((data) => (
+        <CardData key={data.id} data={data} />
+      ))}
+    </div>
+  );
+}
```

Comment:
💊 
위의 코멘트에 제가 `data` 보단 `linkList` 를 사용하면 좋을 것 같다는 코멘트를 남겼었어요.
`data.map((data) => ...)` 부분에서 내부의 data는 외부에서 받은 data와 같은 이름을 사용하고 있기에 코드의 가독성을 저하시킬 수 있으며, 혼동을 일으킬 여지가 있군요 😢 

how about
```suggestion
export function MainSection({ linkList }) { // linkList 또는 links
  return (
    <div className="main-website-section">
      {linkList.map((link) => (
        <CardData key={link.id} data={link} />
      ))}
    </div>
  );
}
```